integrations/terraform/envs/portal/providers.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = "1.9.2"
+  required_version = "~> 1.9.2"
```

Comment:
Best to keep this fixed, to make sure everyone is on the same terraform version.

Reply:
Pinned the Terraform version again with latest commit.